modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -0,0 +1,32 @@
+[id="compatibility-support-matrix_{context}"]
+= Compatibility and support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
```

Comment:
same comments as earlier in doc https://github.com/openshift/openshift-docs/pull/37369/files#r741943346

Reply:
Updated. Thanks.